modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -345,6 +345,7 @@ There are a number of recommended practices related to clustering. These include
 * Use distributed stores such as those used with the VM or JMS connector – these stores are available to an entire cluster. This is preferable to the non-distributed stores used with connectors such as File, FTP, and JDBC – these stores are read by a single node at a time.
 * Use the VM connector to get optimal performance. Use the JMS connector for applications where data needs to be saved after the entire cluster exits.
 * Implement reliability patterns to create high reliability applications.
+* In HA cluster mode, all Object Store content, both in-memory and persistent (as defined by the `persistent` parameter), is stored in the distributed memory grid. As long as the cluster maintains quorum, this data survives application redeploys or restarts. To fully clear the Object Store content, all cluster nodes must be stopped before restarting. For scenarios where data persistence is required even after a full cluster shutdown, configure the Object Store to use a JDBC-based store instead of the memory grid. See xref:mule-high-availability-ha-clusters.adoc#clustering-best-practices[Object Store Persistence for details].
```

Comment:
```suggestion
* In HA cluster mode, all Object Store content, both in-memory and persistent (as defined by the `persistent` parameter), is stored in the distributed memory grid. As long as the cluster maintains quorum, this data survives application redeploys or restarts. To fully clear the Object Store content, all cluster nodes must be stopped before restarting. For scenarios where data persistence is required even after a full cluster shutdown, configure the Object Store to use a JDBC-based store instead of the memory grid. See xref:creating-and-managing-a-cluster-manually.adoc#object-store-persistence[Object Store Persistence].
```